site/get-started.markdown

```diff
@@ -99,7 +99,7 @@ Please look at my favorite odd numbers: [11,13,15,17,19]
 
 ## Participate in the community
 
-By participating in the Haskell community you be able to ask for help and learn about new developments in the Haskell ecosystem. Some of the most popular places to interact with the community are:
+By participating in the Haskell community, you will be able to ask for help and learn about new developments in the Haskell ecosystem. Some of the most popular places to interact with the community are:
```

Comment:
```suggestion
By participating in the Haskell community you will be able to ask for help and learn about new developments in the Haskell ecosystem. Some of the most popular places to interact with the community are:
```

Reply:
I think adding the comma is not correct, actually, though that's based on intuition rather than my knowledge of precise grammatical rules. Anyway, it's better than what's already there.